src/components/players/player/player.tsx

```diff
@@ -1,27 +1,100 @@
-import { type FC } from "react";
-import { Avatar, Stack, Typography } from "@mui/material";
+import React, { type FC, useEffect, useState } from "react";
+import { Stack, Typography } from "@mui/material";
 
 import { styles } from "./styles.js";
 import { Skeleton } from "./skeleton.jsx";
+import { type Character, type MatchStage } from "~/types/index.js";
+import { text } from "~/assets/text/index.js";
+import { CharacterAvatar } from "~/components/character-avatar/index.js";
 
 interface Props {
-  color?: string;
-  characterName: string;
+  character: Character;
+  selected?: boolean;
+  isLocalPlayer?: boolean;
+  hasGuessed?: boolean;
+  isBot?: boolean;
+  onSelectPlayer?: () => void;
+  stage?: MatchStage;
 }
 
-export const Player: FC<Props> = ({ color = "blue", characterName }) => {
-  // TODO: Fix "src" after svg's set up
-  const src = "../images/svg/alien.svg";
+export const Player: FC<Props> = ({
+  character,
+  isLocalPlayer = false,
+  selected,
+  hasGuessed,
+  isBot,
+  onSelectPlayer,
+  stage = "chat",
+}) => {
+  const { name, color } = character;
   const textColor = `${color}.dark`;
+  const [isBotSelect, setIsBotSelect] = useState<boolean | undefined>(false);
 
-  if (!characterName) return <Skeleton />;
+  useEffect(() => {
+    if (stage === "voting" && selected) {
+      setIsBotSelect(true);
+    } else {
+      setIsBotSelect(false);
+    }
+  }, [selected, stage]);
```

Comment:
Just make it `const isBotSelect = stage === "voting" && selected`

Reply:
No need to make it a state var since you are computing it from other values

Reply:
wouldn't it be `isBotSelected`?